project/Deps.scala

```diff
@@ -105,7 +105,6 @@ object Deps {
   )
 
   val coreTest = List(
-    Test.bitcoinj,
```

Comment:
The other thing you should delete in this file is the `val bitcoinj = ...` in the `Test` object